store/postgres/user.go

```diff
@@ -517,3 +533,439 @@ func (us *UserStore) DeleteAccount(ctx context.Context, uid string) error {
 
 	return nil
 }
+
+func (us *UserStore) GetUserProfilePicture(ctx context.Context, uid string) (string, error) {
+	psqlStatement := `SELECT picture from PERSON WHERE id = $1`
+
+	res := us.db.QueryRow(psqlStatement, uid)
```

Comment:
whenerver possible, use the context-ed version. `QueryRowContext`, `QueryContext`, `ExecContext`, etc. It makes it easier if in future we want to alleviate context timeout or add context value used for tagging and debugging